addons/kommander/1.x/kommander-11.yaml

```
@@ -41,7 +41,7 @@ spec:
   chartReference:
     chart: kommander
     repo: https://mesosphere.github.io/charts/stable
-    version: 0.4.8
+    version: 0.4.9
```

```suggestion
    version: 0.4.10
```
I think you could save some time by bumping it to the latest version here.

---

Let's bump this and then I'm :+1: for this change.